integration/servicenow-integration/src/main/java/se/inera/intyg/webcert/integration/servicenow/dto/OrganizationRequest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.integration.servicenow.dto;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.List;
+import se.inera.intyg.webcert.integration.servicenow.dto.OrganizationRequest.OrganizationRequestBuilder;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonDeserialize(builder = OrganizationRequestBuilder.class)
+@Value
+@Builder
+public class OrganizationRequest {
```

Comment:
This class does not follow Inera-codestyle (notice the indentation)... double check any changed classes so they are formatted correct.

Reply:
Corrected this class and checked the others. Done.